addons/web/static/src/core/utils/hooks.js

```diff
@@ -142,9 +142,11 @@ export const SERVICES_METADATA = {};
 export function useService(serviceName) {
     const component = useComponent();
     const { services } = component.env;
-    if (!(serviceName in services)) {
-        throw new Error(`Service ${serviceName} is not available`);
-    }
+    setTimeout(() => {
+        if (!(serviceName in services)) {
+            throw new Error(`Service ${serviceName} is not available`);
+        }
+    }, 0);
```

Comment:
It could be:
```js
new Promise((_, reject) => {
        setTimeout(() => {
            if (!(serviceName in services)) {
                reject(new Error(`Service ${serviceName} is not available`));
            }
        }, 0);
    });
```
to handle errors asynchronously. Probably it's needed in the test.